spec/activefedora/association_spec.rb

```diff
@@ -37,6 +40,20 @@ class Image < ActiveFedora::Base
       subject { reloaded.generic_files }
       it { is_expected.to eq [generic_file2, generic_file1] }
 
+      it "should return an updated array of generic_files" do
+        current_generic_files = image.generic_files.container.to_a
+        new_generic_files = current_generic_files + [generic_file3]
+        image.generic_files = new_generic_files
+        expect(image.generic_files).to eq [generic_file2, generic_file1, generic_file3]
+      end
+
+      it "should return an updated array of generic_files when generic_files is empty" do
+        current_generic_files = image2.generic_files.container.to_a
+        new_generic_files = current_generic_files + [generic_file3]
+        image.generic_files = new_generic_files
+        expect(image2.generic_files).to eq [generic_file3]
```

Comment:
@elrayle I don't understand what this test is supposed to do. You add a file to one image, and expect it to show up in another?